<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="labeld">
   <property name="geometry">
    <rect>
     <x>728</x>
     <y>30</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>连接频率(分)：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit4">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>30</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>30</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labela">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Server：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelb">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Port：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit3">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>30</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelc">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>30</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>IE刷新频率(分)：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="flushbutton">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>30</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>1281</width>
     <height>651</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>911</width>
     <height>0</height>
    </size>
   </property>
   <property name="rowCount">
    <number>60</number>
   </property>
   <property name="columnCount">
    <number>10</number>
   </property>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <item row="0" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="0" column="2">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="0" column="3">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="0" column="4">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="0" column="5">
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
